telegram/ext/_callbackqueryhandler.py

```diff
@@ -55,6 +55,9 @@ class CallbackQueryHandler(BaseHandler[Update, CCT]):
         When setting :paramref:`block` to :obj:`False`, you cannot rely on adding custom
         attributes to :class:`telegram.ext.CallbackContext`. See its docs for more info.
 
+    .. seealso:: `Types of Handlers <https://github.com/\
+        python-telegram-bot/python-telegram-bot/wiki/Types-of-Handlers>`_,
+
```

Comment:
that page is rather generic as well - maybe just link to it from `BaseHandler`

Reply:
removing links to Types of Handlers wiki page from `CallbackQueryHandler`, `MessageHandler`, `CommandHandler`